client/samples/lib/sample/vcenter/vm/hardware/CpuConfiguration/cpu_configuration.rb

```diff
@@ -0,0 +1,95 @@
+# Copyright 2014-2017 VMware, Inc.  All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+require 'sample/framework'
+require 'vapi'
+require 'com/vmware/vcenter/vapi'
+require 'sample/vcenter/helper/vm_helper'
+
+class CPU_Configuration < SampleBase
```

Comment:
There shouldn't be underscores in class names. This should be `CPUConfiguration`.